inst/syms.m

```diff
@@ -58,6 +58,49 @@
 %%
 %% Called without arguments, @code{syms} displays a list of
 %% all symbolic functions defined in the current workspace.
+%% @example
+%% @group
+%% syms x y z
+%% syms
+%%   @print{} Symbolic variables in current scope:
+%%   @print{}   x
+%%   @print{}   y
+%%   @print{}   z
+%% @end group
+%% @end example
+%%
+%% Using @code{syms} to create a new symbol with different assumptions
+%% does not automatically replace instances of the old symbol in the
+%% caller's workspace.  For example suppose we make an absolute value
+%% expression:
+%% @example
+%% @group
+%% syms x
+%% f = abs (x)
+%%   @result{} f = (sym) │x│
+%% @end group
+%% @end example
+%% If we make a new positive @code{x}, @code{f} still contains the old
+%% symbol:
+%% @example
+%% @group
+%% syms x positive
+%% simplify (f)
+%%   @result{} (sym) │x│
+%% @end group
+%% @end example
+%% Note: this behaviour is slightly different from Matlab's Symbolic
+%% Math Toolbox, which does change existing symbols.
+%% If that behaviour is desired, @pxref{@@sym/assume} which modifies
+%% existing expressions in the caller's workspace:
+%% @example
+%% @group
+%% syms x
+%% assume x positive
+%% f
+%%   @result{} f = (sym) x
+%% @end group
+%% @end example
```

Comment:
Should we file an issue regarding SMT compatibility? Otherwise LGTM!

Reply:
Its documented here...  I don't really intend to change it (that `|x|` example is kind of scary to me!)  But go for it if you want.

Thanks for the review.